content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -190,6 +190,8 @@ The following manifest defines an Ingress that sends traffic to your Service via
 
    {{< note >}}If you are running Minikube locally, use `minikube ip` to get the external IP. The IP address displayed within the ingress list will be the internal IP.{{< /note >}}
 
+   {{< note >}}If you are running Minikube locally on Mac, use `127.0.0.1` and run `minikube tunnel`.{{< /note >}}
```

Comment:
Could we use a [tab set](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) to indicate the appropriate local steps based on common configurations with different operating systems?